docs/security-migration.md

```diff
@@ -185,11 +193,12 @@ To ensure all the scans don't run at the same time, please change the time of th
 
 #### Outputs
 
-Veracode processes the reports, which are ultimately made available in the [Developer Portal](https://developer-portal.hmpps.service.justice.gov.uk/reports/veracode).
+Veracode processes the reports, which are ultimately made available in
+the [Developer Portal](https://developer-portal.hmpps.service.justice.gov.uk/reports/veracode).
 
 If the workfow fails, Github sends a slack message and an email to notify users.
 
-### Automation
+## Automation
```

Comment:
moved up a level as I didn't see it originally until I'd already written the script